tools/gvisor-smoke-test.sh

```diff
@@ -42,4 +42,17 @@ else
   install -m555 "$GVISOR_VMLINUX_PATH" "$workdir/kernel/vmlinux"
 fi
 
+set +e
 sudo -E ./bin/syz-manager -config "$workdir/config" --mode smoke-test
+return_code="$?"
+set -e
+
+if [[ "$return_code" -ne 0 ]] && [[ "$(ls -1 "$workdir/logs" | wc -l)" -gt 0 ]]; then
+        for log_file in "$workdir/logs/"*; do
+                echo "-------- Log: $log_file --------" >&2
```

Comment:
">&2" for each log message doesn't look nice and if we want to print all message to stderr, it is better to redirect a script stdout to stderr. It can be done with adding the "exec 1>&2" as the beginning of the script.